src/helpers/TargetingPairs.js

```diff
@@ -31,24 +31,48 @@ var TargetingPairs = {
     return (tagMeta.tags || '').split(',');
   },
 
+  /**
+  * Gets index of article position in reading list
+  *
+  * @param HTMLElement adContainer the ad container currently being filled
+  * @param object kinjaMeta kinja-specific post meta
+  */
+  getArticlePosition: function (adContainer, kinjaMeta) {
+    if (document && adContainer && kinjaMeta.pageType === "permalink" && !(kinjaMeta.post && kinjaMeta.post.isFeatured)) {
+      var readingListPostIds = Array.from(document.getElementsByClassName('js_reading-list-item')).map(post => post.dataset.postId);
+      
+      if (readingListPostIds.length > 0) {
+        var readingListItem = adContainer.closest('.js_reading-list-item');
+        if (readingListItem) {
+          var currentPostId = readingListItem.dataset.postId;
+          return readingListPostIds.indexOf(currentPostId) + 1;
+        }
+      }
+    }
+
+    return 'none';
+  },
+
   /**
    * Constructs associative array of targeting pairs from post meta
    *
    * @param Window scope the window to use for the kinja meta info.
    *
    */
-  buildTargetingPairs: function (scope, positionTargeting) {
+  buildTargetingPairs: function (scope, positionTargeting, adContainer) {
     var kinjaMeta = scope.kinja.meta,
       post = scope.kinja.postMeta || {},
       content = scope.kinja.postContentRatings || [],
       experimentVariation = Experiments.getExperimentVariation(scope),
       experimentId = Experiments.getExperimentId(scope),
       socialTargeting, key,
-      targeting = {};
+      targeting = {},
+      articlePosition = TargetingPairs.getArticlePosition(adContainer, kinjaMeta);
 
     // Begin slot level params AKA 'scp'
     targeting.slotOptions = {
       // Standard targeting options
+      articlePosition,
```

Comment:
This needs to actually be `article_position` - we need it to match up with the same key that was there before and that's defined in DFP

Reply:
🙈 camelcase and snakecase in same object

Reply:
but 👍